custom_components/philips_airpurifier_coap/const.py

```diff
@@ -110,6 +110,7 @@ class FanModel(StrEnum):
     AC0850_20 = "AC0850/20 AWS_Philips_AIR"
     AC0850_20C = "AC0850/20 AWS_Philips_AIR_Combo"
     AC0850_31 = "AC0850/31"
+    AC0850_81 = "AC0850/81 AWS_Philips_AIR_Combo"
```

Comment:
Is the firmware part of the model name needed? Normally, this is only needed if there is another AC0850/81 with a different firmware. I suggest to eliminate this and just use AC0850/81 to recognize this model until someone complains.